### PR TITLE
Kb/public members

### PR DESCRIPTION
**Pull request by kburnell**
- did all the things
- still need to get a really paragraph for the intro.

### CODE REVIEW

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/members**](https://that-website-git-kb-public-members.that-conference.now.sh/members) | [**/partners**](https://that-website-git-kb-public-members.that-conference.now.sh/partners) |
|:-:|:-:|
|<a href="https://that-website-git-kb-public-members.that-conference.now.sh/members">Screenshot of /members</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/5YExOS42j_Z7mcjdZKWPRK6yawE_.png">(view full size)</a> |<a href="https://that-website-git-kb-public-members.that-conference.now.sh/partners">Screenshot of /partners</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/D7UvwA93V5N97DzeYUIAT_dA3_M_.png">(view full size)</a> |







Commit d54f1d43b8db93dbf6c5eb600a9ed715e1eb3feb (https://that-website-git-kb-public-members.that-conference.now.sh).

File: components/shared/SquareButton.js [hunk 1]
```diff
@@ -63,22 +63,32 @@ const SquareButton = ({
 
 SquareButton.propTypes = {
   className: PropTypes.string,
-  backgroundColor: PropTypes.string.isRequired,
```
Comment (saragibby) [inline components/shared/SquareButton.js:66]: Thanks for getting this fixed! Trying to get this filled in when I touch a component, sometimes copy/paste fails you 😂

File: components/shared/SkeletonLoader.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (saragibby) [inline components/shared/SkeletonLoader.js:1]: Nice call to pull this into it's own component. Great start to help us blow this out into skeleton loaders we can use on all the things.

File: pages/members.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+  &:focus {
+    outline: unset;
+  }
+
+  ${below.small`
+    width: 100%;
+  `};
+`;
+
+let fetchingMore = false;
```
Comment (saragibby) [inline pages/members.js:87]: I would put this in state.

File: pages/members.js [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+  );
+  if (loading && !fetchingMore) return <LoadingIndicator />;
+
+  if (error) {
+    throw new Error(error);
+  }
+
+  const loadingMoreMembers = networkStatus === 3;
+
+  let { members, cursor } = data.members.members;
```
Comment (saragibby) [inline pages/members.js:105]: I would use state for these as well. I also bet Apollo is doing come caching of the results within it's store. Did you by chance play with that at all? Or happen to look at the `@connection` directive at all (https://www.apollographql.com/docs/react/data/pagination/#the-connection-directive)?

Comment (kburnell) [reply pages/members.js:105]: nope, looked at how to know when loading on fetch more and this is what the people said to do.

**Pull request merged**